draft-ietf-quic-transport.md

```diff
@@ -4549,13 +4549,9 @@ packets.
 
 After a client receives a Retry packet, 0-RTT packets are likely to have been
 lost or discarded by the server.  A client SHOULD attempt to resend data in
-0-RTT packets after it sends a new Initial packet.
-
-A client MUST NOT reset the packet number it uses for 0-RTT packets, since the
-keys used to protect 0-RTT packets will not change as a result of responding to
-a Retry packet.  Sending packets with the same packet number in that case is
-likely to compromise the packet protection for all 0-RTT packets because the
-same key and nonce could be used to protect different content.
+0-RTT packets after it sends a new Initial packet.  New packet numbers MUST be
```

Comment:
```suggestion
0-RTT packets after it sends a new Initial packet.  New larger packet numbers MUST be
```

Reply:
I wasn't 100% sure what new meant, so I think adding larger avoids any potential confusion.

Reply:
I think that the "reusing packet numbers" is clear enough.